### PR TITLE
Improvement of detection logic

### PR DESCRIPTION
**Pull request by xFFninja**
### Summary of the Pull Request
Improvement of detection logic

+ additional fieldname AccessList
+detection values

### Detailed Description of the Pull Request / Additional Comments
`Accesses `field values may appear as code. The string value appears in EventViewer, but different SIEMs can store the original value as it is stored in xml (e.g. in Microsoft Sentinel this field is not extracted by default data connector and user should extract it during query time to be able to use it in detection). Also the field name in xml is `AccessList `but not Accesses.

### Example Log Event
https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4656
image
image

### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

**Comment by nasbench** [comment]
Hey, thanks for the contribution.

I agree with you as there was an error during the merger of this rule.

Just le me add a couple of things we don't need other fields or graphical representation. That's the job of the backend to map to other fields or interpret values.

Since your branch is protected I can't make the changes. So please update the detection to look like this.

```yml
selection:
        EventID:
            - 4656
            - 4663
        ProcessName|endswith: '\OUTLOOK.EXE'
        # Example: \REGISTRY\MACHINE\SYSTEM\CurrentControlSet001\Services\WebClient\NetworkProvider
        ObjectName|contains|all:
            - '\REGISTRY\MACHINE\SYSTEM'
            - 'Services\'
        ObjectName|endswith:
            - 'WebClient\NetworkProvider'
            - 'LanmanWorkstation\NetworkProvider'
        AccessList:  '%%4416' # "Query key value"
    condition: selection
```

File: rules/windows/builtin/security/win_security_exploit_cve_2023_23397_outlook_remote_file_query.yml [hunk 1]
```diff
@@ -20,15 +20,15 @@ detection:
...
         # Example: \REGISTRY\MACHINE\SYSTEM\CurrentControlSet001\Services\WebClient\NetworkProvider
         ObjectName|contains|all:
             - '\REGISTRY\MACHINE\SYSTEM'
             - 'Services\'
         ObjectName|endswith:
             - 'WebClient\NetworkProvider'
             - 'LanmanWorkstation\NetworkProvider'
-    condition: selection
+        AccessList|contains:  '%%4416' # "Query key value"
+condition: selection
```
Comment (nasbench) [inline rules/windows/builtin/security/win_security_exploit_cve_2023_23397_outlook_remote_file_query.yml:31]:
```suggestion
    condition: selection
```

**Review by nasbench — APPROVED** [review]
Also update the modified date.

File: rules/windows/builtin/security/win_security_exploit_cve_2023_23397_outlook_remote_file_query.yml [hunk 2]
```diff
@@ -20,14 +20,14 @@ detection:
...
         ProcessName|endswith: '\OUTLOOK.EXE'
-        Accesses|contains: 'Query key value'
         # Example: \REGISTRY\MACHINE\SYSTEM\CurrentControlSet001\Services\WebClient\NetworkProvider
         ObjectName|contains|all:
             - '\REGISTRY\MACHINE\SYSTEM'
             - 'Services\'
         ObjectName|endswith:
             - 'WebClient\NetworkProvider'
             - 'LanmanWorkstation\NetworkProvider'
+        AccessList|contains:  '%%4416' # "Query key value"
```
Comment (nasbench) [inline rules/windows/builtin/security/win_security_exploit_cve_2023_23397_outlook_remote_file_query.yml:30]:
```suggestion
        AccessList|contains: '%%4416' # "Query key value"
```

**Comment by nasbench** [comment]
@xFFninja to make the review process easier for both of us. Next time please unprotect your branch :)

**Comment by xFFninja** [comment]
> @xFFninja to make the review process easier for both of us. Next time please unprotect your branch :)

already done ;)

**Pull request merged**